src/features/infinite-scroll/js/infinite-scroll.js

```diff
@@ -22,6 +22,16 @@
    */
   module.service('uiGridInfiniteScrollService', ['gridUtil', '$compile', '$timeout', 'uiGridConstants', 'ScrollEvent', '$q', function (gridUtil, $compile, $timeout, uiGridConstants, ScrollEvent, $q) {
 
+
+    // not sure where to put this util function @vance
```

Comment:
ehh. splitting time between a fortune 500 a startup and marketing my own side-app at the moment. I don't know the next time I will have the headspace to compile this again, christmas time?